lsp/src/Language/LSP/Server/Core.hs

```diff
@@ -304,8 +304,8 @@ data ServerDefinition config = forall m a.
       --
       -- @
       --  ServerDefinition { ...
-      --  , doInitialize = \env _req -> pure $ Right env
-      --  , interpretHandler = \env -> Iso
+      --  , doInitialize = \\env _req -> pure $ Right env
```

Comment:
Special characters (like `\`) need to be escaped in @ code blocks: https://haskell-haddock.readthedocs.io/en/latest/markup.html#code-blocks